Jenkinsfile

```diff
@@ -111,6 +111,12 @@ podTemplate(name: podName,
                         ttyEnabled: true,
                         command: 'cat',
                         privileged: true,
+                        workingDir: '/workDir'),
+                containerTemplate(name: 'ostree-boot-image',
+                        image: DOCKER_REPO_URL + '/' + OPENSHIFT_NAMESPACE + '/ostree-boot-image:' + SLAVE_TAG,
+                        ttyEnabled: true,
+                        command: '/usr/sbin/init',
```

Comment:
should be 'cat' for command

Reply:
pretty sure this has to be /usr/sbin/init or the libvirtd service won't start